rasa/shared/core/training_data/visualization.py

```diff
@@ -456,6 +479,33 @@ def visualize_neighborhood(
             if isinstance(el, UserUttered):
                 message = el.parse_data
```

Comment:
i don't think the check made any sense there. i don't when in UserUttered, you would have multiple slots in a row. That's not the same event.